docs/source/guides/dockerfile_generation.rst

```diff
@@ -45,20 +45,18 @@ This allows BentoML to enable users to fully customize how to structure a Bento'
 
    So, make sure you know what you are doing if you are trying out something like above.
 
-In addition to Jinja, BentoML also uses `Docker Buildx <https://docs.docker.com/desktop/multi-arch/>`_, which enables users to build Bentos that support
-multiple architectures. Under the hood the generated Dockefile leverage
-`buildkit <https://github.com/moby/buildkit>`_, which enables some advanced `dockerfile features <https://github.com/moby/buildkit/blob/master/frontend/dockerfile/docs/syntax.md#dockerfile-frontend-syntaxes>`_.
+In addition to Jinja, BentoML also take advantage of `buildkit <https://github.com/moby/buildkit>`_, which enables some advanced `containerization features <https://github.com/moby/buildkit/blob/master/frontend/dockerfile/docs/syntax.md#dockerfile-frontend-syntaxes>`_, such
+as better layer caching, multiple architecture support, etc.
 
-.. note::
+.. admonition:: supporting different ``buildkit`` frontend
```

Comment:
let's remove this section since it's not relevant to most users and we don't currently support it